### ISSUE TITLE
Add "virtual environments" section in navigation bar

### ISSUE THREAD

**Issue by damko** [freedomofpress/securedrop-docs#219]
## Describe the change
I propose to move these pages from the navigation section "Developer documentation" to a new section above called "Virtual Environments": 

* Virtual environments: Servers 
* Virtual environments: Admin Workstation
* Virtual environments: Using Qubes
* Portable SecureDrop Demo

Eventually the page files could be moved to a separate folder like "venvs" but, what's important to me is to have a separate section like this:

![securedrop_menu_virtual_environments](https://user-images.githubusercontent.com/1107588/118206835-a0f36600-b463-11eb-85bb-f1e5297033da.png)

## How will this impact users?
1. I think that a less crowded "Developer documentation" section would be beneficial for everyone
2. New administrators won't have to look through all the documentation before finding what they need to have a virtualized environment for evaluation purposes. At the same time developers won't have any difficulty in finding what they need even if it's not in "their" section. Win-win

## User Research Evidence
I'm new to Securedrop at it was hard for me to navigate the documentation. Only after two days with my nose in the documentation I realized that a portable setup is described: I simply couldn't see it.

**Comment by eloquence** [comment]
@damko These virtual environments are only intended for development, so we do want to keep them clearly situated in the dev docs. That said, the information architecture of the development section of the docs definitely needs improvement.

**Comment by damko** [comment]
@eloquence yes, on a second analysis, I reached your same conclusion. It should stay in the development section  however I still felt the need for a "hit the ground running" kind of tutorial which would satisfy the needs of those willing to evaluate SecureDrop so I wrote it (currently ~70% of the draft). I'm writing it while I build my own "evaluation environment". When completed I'll submit it as a ticket so that you guys can have a look at it and understand if it's something desirable.
